### PR TITLE
Create docs.md

### PR DESCRIPTION
**Pull request by AsmaKacem1**
Brief documentation in Arabic about the translation process of the project in Crowdin.

### CODE REVIEW

**Comment by AsmaKacem1** [comment]
Brief documentation in Arabic about the translation process of the project in Crowdin.

**Comment by BatoolMM** [comment]
Thank you @AsmaKacem1. I'll provide a review soon!

File: docs.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+ تجربتي مع كراودن (Crowdin)
```
Comment (BatoolMM) [inline docs.md]:
```suggestion
 ## تجربتي مع كراودن (Crowdin)
```

Comment (BatoolMM) [reply docs.md]: I liked reading about your experience using Crowdin but the file is plain text without any formatting. You need to use Markdown syntax to format the headings, bulletin points, ...etc.

For example, the header should start with #.  [Here are more details about Markdown syntax](https://www.markdownguide.org/basic-syntax/). Let me know if you have questions!

**Review by BatoolMM — CHANGES_REQUESTED** [review]

File: docs.md (hunk 1 above)
Comment (AsmaKacem1) [reply docs.md]: Thank you very much @BatoolMM for your feedback. 
I will make the necessary changes.

**Comment by AsmaKacem1** [comment]
I have changed this file to a Markdown file format @BatoolMM. 
would you please provide me with your feedback.

**Comment by BatoolMM** [comment]
It looks so beautiful now - can you add more instructions on how to use Crowdin? Where do you click, how to start and how do you use in Crowdin editor?

**Comment by AsmaKacem1** [comment]
Thank you for your feedback. 
yes,i will add more instructions.

**Comment by BatoolMM** [comment]
@AsmaKacem1 don't forget to record your contribution in [Outreachy](https://www.outreachy.org/outreachy-may-2022-internship-round/communities/open-science-community-saudi-arabia-oscsa/localisation-of-open-educational-resources-of-open/contributions/). Let me know if you have any question about that!

File: docs.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+موقع **كراودن (Crowdin)** هو موقع للترجمة و التعريب معتمد لدى الكثير من الشركات منها **ميكروسوفت** و **غوغل** حيث تقوم  بوضع الملفات الخاصة بمواقعها او برامجها و طلب تعريبها من الناس.  
+
+هذا الموقع سهل الإستعمال حيث انك تقوم بالدخول و اختيار احد المشاريع والبدء في تعريبها و الشيء الرائع انه يدعم اللغة العربية و قد تم تعريب الكثير من تطبيقات و المواقع بواسطته.   
+
+
+## ٢. بداياتي :          
+
+
+كانت البداية صعبة قليلا ،نظرا بكونها أول مشاركة لي في عالم **المصادر المفتوحة (open source)**.فكانت خطوتي الأولى فهم أساسيات المصادر المفتوحة و  طريقة استعمالها و قد تبين لي مدى أهميتها في كل المجالات. بعد ذلك بدأت في قراءة تفاصيل المشاركة في هذا المشروع.
```
Comment (BatoolMM) [inline docs.md]:
```suggestion
كانت البداية صعبة قليلا ،نظرا بكونها أول مشاركة لي في عالم **المصادر المفتوحة (open source)** فكانت خطوتي الأولى فهم أساسيات المصادر المفتوحة و  طريقة استعمالها و قد تبين لي مدى أهميتها في كل المجالات. بعد ذلك بدأت في قراءة تفاصيل المشاركة في هذا المشروع.   
```

File: docs.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+هذا الموقع سهل الإستعمال حيث انك تقوم بالدخول و اختيار احد المشاريع والبدء في تعريبها و الشيء الرائع انه يدعم اللغة العربية و قد تم تعريب الكثير من تطبيقات و المواقع بواسطته.   
+
+
+## ٢. بداياتي :          
+
+
+كانت البداية صعبة قليلا ،نظرا بكونها أول مشاركة لي في عالم **المصادر المفتوحة (open source)**.فكانت خطوتي الأولى فهم أساسيات المصادر المفتوحة و  طريقة استعمالها و قد تبين لي مدى أهميتها في كل المجالات. بعد ذلك بدأت في قراءة تفاصيل المشاركة في هذا المشروع.   
+
+في مرحلة ثانية قمت بإختيار ملف و بدأت  في الترجمة . لم تكن التجربة في منتهى السهولة ،بل واجهت الكثير من الصعوبات لكن لم أستسلم لذلك ، و حاولت طرح التساؤلات فقامت المرشدة (mentor) و المجموعة (community) بمساعدتي .أهم شيء لا تخجل و قم بطرح إستفساراتك.
```
Comment (BatoolMM) [inline docs.md]:
```suggestion
في مرحلة ثانية قمت بإختيار ملف و بدأت  في الترجمة، لم تكن التجربة في منتهى السهولة ، بل واجهت الكثير من الصعوبات لكن لم أستسلم لذلك ، و حاولت طرح التساؤلات فقامت المرشدة (mentor) و المجموعة (community) بمساعدتي .أهم شيء لا تخجل و قم بطرح إستفساراتك.   
```

File: docs.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+كانت البداية صعبة قليلا ،نظرا بكونها أول مشاركة لي في عالم **المصادر المفتوحة (open source)**.فكانت خطوتي الأولى فهم أساسيات المصادر المفتوحة و  طريقة استعمالها و قد تبين لي مدى أهميتها في كل المجالات. بعد ذلك بدأت في قراءة تفاصيل المشاركة في هذا المشروع.   
+
+في مرحلة ثانية قمت بإختيار ملف و بدأت  في الترجمة . لم تكن التجربة في منتهى السهولة ،بل واجهت الكثير من الصعوبات لكن لم أستسلم لذلك ، و حاولت طرح التساؤلات فقامت المرشدة (mentor) و المجموعة (community) بمساعدتي .أهم شيء لا تخجل و قم بطرح إستفساراتك.   
+
+
+## ٣. المشاكل التي واجهتني و كيف تجاوزتها :                       
+
+
+ * لا تعتمد علي غوغل ترادكشن أو **آلة كراودين (Crowdin)** في الترجمة فهذه الطريقة خاطئة حيث أنها تقوم بالترجمة كلمة بكلمة  فلا تراعي سياق  الجملة و معناها ،و في أغلب الأوقات لا تطابق المعنى الأصلي للجملة.إنما تكمن الطريقة الصحيحة في قراءة الجملة و التمعن في معناها وترجمتها حسب السياق بكلماتي و تعابيري مع الإستعانة بغوغل ترادكشن في بعض الأحيان لترجمة كلمة معينة.
```
Comment (BatoolMM) [inline docs.md]:
```suggestion
 * لا تعتمد علي غوغل ترانسليشن أو **آلة كراودين (Crowdin)** في الترجمة فهذه الطريقة خاطئة حيث أنها تقوم بالترجمة كلمة بكلمة  فلا تراعي سياق  الجملة و معناها، و في أغلب الأوقات لا تطابق المعنى الأصلي للجملة، إنما تكمن الطريقة الصحيحة في قراءة الجملة و التمعن في معناها وترجمتها حسب السياق بكلماتي و تعابيري مع الإستعانة بغوغل ترادكشن في بعض الأحيان لترجمة كلمة معينة.   
```

File: docs.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+* لا تقم بترجمة الجمل التي تحتوي علي **شفرة (code)**.مثال:   
+
+![Capture d’écran 2022-04-09 205113](https://user-images.githubusercontent.com/73066984/162591145-71b0d72b-84dd-4cba-9151-0fe7d9f3f2fb.png)
+
+
+![Capture d’écran 2022-04-09 205634](https://user-images.githubusercontent.com/73066984/162591150-dbac7f7a-b210-463e-abc4-8340224e3847.png)
+
+
+* إحذر أخطاء التنقيط. فعند القيام بالترجمة يجب الحرص علي التنقيط. فمثلا **"."** في الإنجليزية تصبح بالعربية **"،"** :
```
Comment (BatoolMM) [inline docs.md]:
```suggestion
* إحذر أخطاء الترقيم فعند القيام بالترجمة يجب الحرص علي التنقيط. فمثلا **"."** في الإنجليزية تصبح بالعربية **"،"** :
```

**Review by BatoolMM — CHANGES_REQUESTED** [review]

**Comment by BatoolMM** [comment]
Hi @AsmaKacem1, I made a few suggestions - if you commit these changes, I'll approve the PR and merge it even if you don't add details about Crowdin!

**Comment by AsmaKacem1** [comment]
> Hi @AsmaKacem1, I made a few suggestions - if you commit these changes, I'll approve the PR and merge it even if you don't add details about Crowdin!

Thank you @BatoolMM,I have committed these changes and I'm working on the details about Crowdin ,I will add them tonight!

**Comment by AsmaKacem1** [comment]
> @AsmaKacem1 don't forget to record your contribution in [Outreachy](https://www.outreachy.org/outreachy-may-2022-internship-round/communities/open-science-community-saudi-arabia-oscsa/localisation-of-open-educational-resources-of-open/contributions/). Let me know if you have any question about that!

Hi @BatoolMM ,In the description section of Record Contributions should I mention the steps I did in the contribution? or there are other things I should mention?

File: docs.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
+# تجربتي مع موقع كراودن (Crowdin)
+
+
+## ١. ماهو موقع كراودن (Crowdin) ؟
+
+موقع **كراودن (Crowdin)** هو موقع للترجمة و التعريب معتمد لدى الكثير من الشركات منها **ميكروسوفت** و **غوغل** حيث تقوم  بوضع الملفات الخاصة بمواقعها او برامجها و طلب تعريبها من الناس.  
+
+هذا الموقع سهل الإستعمال حيث انك تقوم بالدخول و اختيار احد المشاريع والبدء في تعريبها و الشيء الرائع انه يدعم اللغة العربية و قد تم تعريب الكثير من تطبيقات و المواقع بواسطته.   
+
+## ٢. الخطوات لإستعمال كراودين :
```
Comment (BatoolMM) [inline docs.md]:
```suggestion
## ٢. الخطوات لإستعمال كراودن :
```

File: docs.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+* **الخطوة الأولى :** [إنشاء حساب](https://accounts.crowdin.com/turingway/login?continue=%2Fu) على  كراودين ثم الانضمام إلى المشروع.
```
Comment (BatoolMM) [inline docs.md]:
```suggestion
* **الخطوة الأولى :** [إنشاء حساب](https://accounts.crowdin.com/turingway/login?continue=%2Fu) على  كراودن ثم الانضمام إلى المشروع.   
```

File: docs.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+![Capture d’écran 2022-04-09 210655](https://user-images.githubusercontent.com/73066984/162591172-ae02307d-1fa5-4f2b-9367-ba5db9718716.png)
+
+
+## ٥. بعض النصائح :
+
+- يجب أن تقوم بقراءة  الفصل الفرعي من كتاب **تورينج آوي (The Turing Way)** و فهم معناه و ما يناقشه .ثم اختر قسما فرعيا و إفهمه قبل أن تقوم بالترجمة  فمن الأخطاء الشائعة ترجمة الجمل الفردية دون قراءة النص أو فهم السياق.
+- قم بفهم الكلمات التقنية جيدا قبل ترجمتها. 
+- إحسن إختيار الكلمات التي ستضعها في الكلوساري (Glosary)
+- بعد إختيار النص الذي ستقوم بترجمته إحرص علي إختيار **"إذهب لتعديله " (Go to editor)** هكذا ستتمكن من ترجمة النص.
+- قم بطرح الأسئلة و لا تتردد حيث ستكسبك الوقت و الخبرة من عند المجموعة و المرشدة (mentor)
```
Comment (BatoolMM) [inline docs.md]:
```suggestion
- قم بطرح الأسئلة و لا تتردد حيث ستكسبك الوقت و الخبرة من عند المجموعة و المرشدة  بتول المرزوق (mentor)
```

**Review by BatoolMM — APPROVED** [review]
Love your review - I made minor comments to fix some typos ! 

Thank you @AsmaKacem1

**Comment by BatoolMM** [comment]
@AsmaKacem1 when submitting your final contribution through [this link](https://www.outreachy.org/outreachy-may-2022-internship-round/communities/open-science-community-saudi-arabia-oscsa/localisation-of-open-educational-resources-of-open/contributions/) include the steps you followed & what is the timeline to go through all the tasks - and what you anticipate to get out of this project. Please let me know if anything isn't clear!

**Comment by AsmaKacem1** [comment]
> Love your review - I made minor comments to fix some typos !
> 
> Thank you @AsmaKacem1

Thank you so much @BatoolMM  for your support and feedback

**Comment by AsmaKacem1** [comment]
> @AsmaKacem1 when submitting your final contribution through [this link](https://www.outreachy.org/outreachy-may-2022-internship-round/communities/open-science-community-saudi-arabia-oscsa/localisation-of-open-educational-resources-of-open/contributions/) include the steps you followed & what is the timeline to go through all the tasks - and what you anticipate to get out of this project. Please let me know if anything isn't clear!
Capture d’écran 2022-04-18 214147

That will be the proposed timeline during the internship?
what's the plan of the work to accomplish the project?

**Comment by BatoolMM** [comment]
This is the plan as described in the main issue #1 
> Tasks during the internship
> Expands the Glossary and ensures consistency in the Translation
> Collect feedback from the community members about the translation workflow.
> Translate 6 parts/sub-chapters from the Guide for Reproducible Research.
> Review other contributions in the Translation
> Deploy the book with the 6 translated chapters
> Provide documentation about the Workflow in Arabic.

It's up to you to estimate the time for each task - we'll revisit the timeline later and make modifications. Don't stress about it too much! 

Let me know if there's anything else you need!

**Pull request merged**